python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/port-channel-interfaces.j2

```diff
@@ -28,9 +18,42 @@ interface {{ port_channel_interface.name }}
 {%     if port_channel_interface.mtu is arista.avd.defined %}
    mtu {{ port_channel_interface.mtu }}
 {%     endif %}
+{%     if port_channel_interface.logging.event.link_status is arista.avd.defined(true) %}
+   logging event link-status
+{%     elif port_channel_interface.logging.event.link_status is arista.avd.defined(false) %}
+   no logging event link-status
+{%     endif %}
 {%     if port_channel_interface.bgp.session_tracker is arista.avd.defined %}
    bgp session tracker {{ port_channel_interface.bgp.session_tracker }}
 {%     endif %}
+{%     if port_channel_interface.l2_protocol.forwarding_profile is arista.avd.defined %}
```

Comment:
Check for all switchport commands order, there are 2 of each command (deprecated and new).
Please keep both of them together and in-order.

Reply:
Fixed. Thanks.